kubectl-fdb/cmd/remove_process_group.go

```diff
@@ -139,14 +149,22 @@ func replaceProcessGroups(kubeClient client.Client, clusterName string, ids []st
 		}
 		return err
 	}
-	// In this case the user has Pod name specified
+
 	var processGroupIDs []fdbv1beta2.ProcessGroupID
-	if !useProcessGroupID {
+	if processClass != "" { // match against a whole process class, ignore provided ids
+		if len(ids) != 0 {
+			return fmt.Errorf("process identifiers were provided along with a processClass and would be ignored, please only provide one or the other")
```

Comment:
the error handling is currently rather passive, but I felt like being explicit could be helpful here (and below on line 160). Let me know if this is not preferred, or if I should make other cases error more clearly.

Reply:
Sounds reasonable and makes sense to me 👍